R/zzz.R

```diff
@@ -1,16 +1,20 @@
-
 tabpfn <- NULL
 
 .onLoad <- function(...) {
+  pkg <- asNamespace("TabPFN")
+  pkg$check_libomp()
```

Comment:
`check_libomp` currently gives a false-positive for the "correct" libomp, if PyTorch is already loaded. 

We can condition this check to prevent that (no need for `pkg`, the load  hook retains its namespace)
```
if (!(reticulate:::py_available() && reticulate::py_module_available("torch")))
 check_libomp()
```

Reply:
That worked!